<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Frame" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QWidget" name="generateFrame" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QWidget" name="generateButtonWidget" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QPushButton" name="generateButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Constantia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Üret</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="generateLineWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Constantia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Adet Kelime  </string>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <widget class="QComboBox" name="wordNumberComboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="learnedWidget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLabel" name="learnedLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Öğrenilen Kelime Sayısı : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLabel" name="leftLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kalan Kelime Sayısı : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resetWidget" native="true">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
